app/controllers/api/v1/monsters_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Api::V1::MonstersController < ApplicationController
+  before_action :authenticate_user, except: [:index, :show]
+  before_action :authorize_user, except: [:index, :show, :create]
```

Comment:
For readability, can we update these to be _inclusive_ of the actions we want these methods to run?